controllers/nstemplatetier/nstemplatetier_controller_test.go

```diff
@@ -338,6 +316,119 @@ func TestReconcile(t *testing.T) {
 
 }
 
+func TestUpdateNSTemplateTier(t *testing.T) {
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
out of curiosity - why do we need to configure the logger in the unit test?

Reply:
yeah, it was a copy paste left over , removed in https://github.com/codeready-toolchain/host-operator/pull/1130/commits/87ea29c751184e89e7bf476da2c6b26c1ca669b4